en/orm/validation.rst

```diff
@@ -84,13 +84,13 @@ To create a default validation object in your table, create the
         }
     }
 
-The available validation methods and rules come from the ``Validator`` class and
-they are documented in the :ref:`creating-validators` section.
+The available validation methods and rules come from the ``Validator`` class and 
+are documented in the :ref:`creating-validators` section.
 
 .. note::
 
-    Validation objects are mainly meant to be used to validate user input, i.e.
-    forms and any other posted request data.
+    Validation objects are intended primarily for validating user input (i.e.,
```

Comment:
You seem to be a fan of brackets. Are you sure they are an improvement to the language of the book?
Normally they don't really help the flow of the sentences...